### DISCUSSION TITLE
How do I use { } in a suffix?

### DISCUSSION THREAD

**Discussion by Varner184** [Zrips/CMI#8053] [General]
I am trying to make suffixes for my server and I want it to show up as "{Rank}" but {} is being recognized as a placeholder so it won't allow me to use it. I have to use (Rank). Is there a work around so that I can use { }?

**Comment by CloudeLecaw** [comment]
placeholders should be used with %placeholder%
your using cmi ranks?

%cmi_user_rank%
%cmi_user_rank_displayname% – Current rank display name
%cmi_user_rank_name% – Current rank name

if you might got confusee because some placeholders in cmi chat use {}..ignore that. these are just basic placeholder that were inbuild you can still use the %placeholder% in the chat format
